package.json

```diff
@@ -46,6 +46,7 @@
         "core-util-is": "^1.0.2",
         "date-fns": "1.30.1",
         "esbuild-plugin-svg": "0.1.0",
+        "test_esm_pkg": "file:./tools/npm_packages/test_esm_pkg",
```

Comment:
lets put this beside the other `test` target. Also seems like the list is alphabetical.